dvc/command/gc.py

```diff
@@ -100,7 +100,7 @@ def add_parser(subparsers, parent_parser):
         "--all-commits",
         action="store_true",
         default=False,
-        help=argparse.SUPPRESS,
+        help="Keep data files for all git commits.",
```

Comment:
Minor: capitalize Git here and in the previous 2 help strings? 🙂 

And (unrelated sorry) in the one before that maybe say "current workspace (working tree)" since we use that Git term a lot in the core repo and some other help outputs, I think.

> Same in dvc.zsh